### DISCUSSION TITLE
ScriptReference/Overlays.Overlay

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#617] [Notes]
# ScriptReference/Overlays.Overlay

https://docs.unity3d.com/ScriptReference/Overlays.Overlay.html

**Comment by Thundernerd** [comment]
You can easily (and accidentally) hide overlays, thankfully there are a couple of ways to bring them back.

You can bring up your overlay menu by right-clicking on the `Scene` tab and clicking on the `Overlay Menu` option.

That `Overlay Menu` option also has a hotkey, which by default is the back tick ``` ` ```.

**Reply by vertxxyz** [reply]
Noting that this sort of usage information is present on the [Overlays section of the manual](https://docs.unity3d.com/Manual/overlays.html).
